src/sqlparse_fold.erl

```diff
@@ -70,8 +73,10 @@ fold(FType, Fun, Ctx, Lvl, {stmtList, STs})
         fun(ST, {Sql, AccCtx}) ->
             {NewSql, NewAccCtx} = fold(FType, Fun, AccCtx, Lvl, ST),
             {Sql ++
-                if length(Sql) > 0 andalso length(NewSql) > 0 ->
-                    ";" ++ NewSql; true -> NewSql end,
+                if length(Sql) > 0 andalso length(NewSql) > 0
+                    -> ";" ++ NewSql;
```

Comment:
The following coding style for multi-line `if` block is maintained and preferred:
```erlang
...
Line 1: if expression ->
Line 2: [8 spaces] body_expression1,
Line 3: [8 spaces] body_expression2,
...
Line n: [8 spaces] body_expression2;
Line n+1: [4 spaces] true ->
Line n+2: [8 spaces] body_expression1
Line n+3:end
...
```

Reply:
Fixed - I did replace all `if` with `case` since my formatter (IntelliJ IDEA with Erlang plugin) is not very `if` friendly.